File: Lecture4/PythagoreanAngles.c
#include <stdio.h>
#include <math.h>

double toDegrees(double rad) {
    return rad / M_PI * 180;
}

int main(void) {
    double AB = 4;
    double AC = 3;

    double alpha = atan(AC / AB);
    double beta = atan(AB / AC);

    printf("alpha = %.2lf radians, %.2lf degrees\n",
        round(alpha*100)/100.0, round(toDegrees(alpha)*100)/100.0);
    printf("beta = %.2lf radians, %.2lf degrees\n",
        round(beta*100)/100.0, round(toDegrees(beta)*10)/10.0);

    return 0;
}

File: Lecture4/PythagoreanAngles.java
public class PythagoreanAngles {
    public static void main(String[] args ) {
        double AB = 4;
        double AC = 3;

        double alpha = Math.atan(AC / AB);
        double beta = Math.atan(AB / AC);

        System.out.println(
            "alpha = " + Math.round(alpha*100)/100.0 + " radians, " +
            Math.round(Math.toDegrees(alpha)*100)/100.0 + " degrees" );
        System.out.println(
            "beta = " + Math.round(beta*10)/10.0 + " radians, " +
            Math.round(Math.toDegrees(beta)*10)/10.0 + " degrees" );
    }
}
